docs/guides/docs/testnet-migration/beta-3-to-beta-4-migration.mdx

```diff
@@ -0,0 +1,223 @@
+---
+title: Beta 4 Testnet Migration 
+category: Testnet Migration
+parent:
+  label: Guides
+  link: /guides
+---
+
+# Beta 4 Migration
+
+This guide focuses on the transition from the end of tool support for `beta 3` to the start of `beta 4`.
+
+- [Fuelup v0.19.4](https://github.com/FuelLabs/fuelup/releases/tag/v0.19.4)
+- [Sway v0.44.0](https://github.com/FuelLabs/sway/releases/tag/v0.44.0)
+- [TS SDK v0.54.0](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.54.0)
+- [Rust SDK v0.47.0](https://github.com/FuelLabs/fuels-rs/releases/tag/v0.47.0)
+- [Wallet SDK v0.12.2](https://github.com/FuelLabs/fuels-wallet/releases/tag/v0.12.2)
+
+## Fuelup Toolchain
+
+Update to the latest version of `fuelup`:
+
+```sh
+fuelup self update
+```
+
+Install the `beta-4` toolchain:
+
+```sh
+fuelup toolchain install beta-4
+```
+
+Set the `beta-4` toolchain as your default:
+
+```sh
+$ fuelup default beta-4
+default toolchain set to 'beta-4-aarch64-apple-darwin'
+```
+
+## Wallet
+
+Fuel Wallet version `0.11.0` is not compatible with beta 4. For compatibility with beta-4, please download the non-backwards compatible Fuel Wallet version `0.12.2`.
+
+```sh
+npm install @fuel-wallet@0.12.2
```

Comment:
```suggestion
pnpm install @fuel-wallet@0.12.2
```

Wondering if we should start promoting `pnpm` over `npm`.
We have a `pnpm` file in this PR too.